### PR TITLE
Bump up AprilTags version to most recent release

### PR DESCRIPTION
**Pull request by fglinka**
I was working with the Apriltags OroGen task and encountered some segfaults, when enabling refine_poses. Those mostly originate from a bug in the apriltags library which has been fixed since, thus I built the most recent Github release of Apriltags in my workspace which seemed to work well.

I encountered no compilation errors and my test setup of running the apriltags-OroGen-Task worked and did not crash when refine_poses was enabled, as is the case with the current version from 2015.

I also removed the CMake and pkg-config part from the patch as the apriltags-library now provides that out-of-the-box.

Blurb from CI
##########
The CI in use has a time limit for building. Hence, we require a particular branch naming schema for PRs, so that a single package build can be tested.

Use the following schema to trigger the build: `update__<name-of-your-package>`
For instance to add an oroGen package `drivers/orogen/new_driver`, create the following branch to create your PR: `update__drivers/orogen/new_driver`

### CODE REVIEW

**Comment by doudou** [comment]
I think this makes a lot of sense ... 5 years of changes are probably worth it ;-)

Let's wait a bit before we merge, to see if others may have issues with the version bump ...

File: source.yml
Comment (chhtz) [inline source.yml:300]: I guess we can also make this

    github: AprilRobotics/apriltag

And the manifest can be moved to rock.core/manifests/perception/apriltags.xml, instead of adding it via patch.

Comment (fglinka) [reply source.yml:300]: Great suggestion! Do we still want to keep it pinned to a release tag, or do we want to follow their master branch, though?

Comment (doudou) [reply source.yml:300]: Agreed.

Comment (fglinka) [reply source.yml]: Thanks for your feedback! I added a commit which uses the current github master branch and moves the manifest from the patch into the `manifests`-Folder as Christoph suggested.

**Review by chhtz — APPROVED** [review]

**Comment by chhtz** [comment]
@doudou This looked fine to me, but I don't have rights to merge this. Can you merge this, or do we need another review?

**Comment by doudou** [comment]
I think we're good

**Pull request merged**
